### ISSUE TITLE
Revisit trade-off visualisation of edges in small plotting devices

### ISSUE THREAD

**Issue by BAndrews22** [pepijn-devries/ggsankeyfier#14]
Currently, when trying to plot multiple Sankeys at once with facet_grid(), the connectors are much too thin and do not take up the entire space of the node.

**Comment by pepijn-devries** [comment]
Hi @BAndrews22,

I'm not sure if this is a bug report, a feature request or even an issue. It sounds as if you are trying something that is either not feasible or possible. In order to help you our or to prepare a fix (if needed) I need a more detailed report with a [reprex](https://reprex.tidyverse.org/). I'm looking forward to your report.

Cheers,

Pepijn

**Comment by BAndrews22** [comment]
Hi Pepijn,
Thanks a mil for solving my other issue with printing to PDF.
I can confirm that this issue is related, the area of the facets gets too narrow. 
By resizing the plot area, also the facetted plot works. But one really has to fiddle with the width and height to get it  work.

I have done the same plot with the ggsankey package and it seems much less sensitive to the aspect ratio. 
However, the output of your package looks better and its syntax is cleaner, so I will rather adjust the aspect ratio.

Just for completeness, below a repex. Depending on the size of the plot window, the output looks correct or not.
Many thanks!

```
library(ggplot2)
library(ggsankeyfier)
theme_set(theme_light())
data("ecosystem_services")

es_sub <-
  ecosystem_services |>
  subset(RCSES > quantile(RCSES, 0.99)) |>
  pivot_stages_longer(c("activity_realm", "biotic_realm", "service_section"), "RCSES", "service_section")

es2 <- cbind(es_sub, pl=rep(1:10, each=nrow(es_sub)))
pos <- position_sankey(v_space = "auto", order = "ascending")

ggplot(es2, aes(x = stage, y = RCSES, group = node, connector = connector, edge_id = edge_id)) + 
  geom_sankeyedge(aes(fill = service_section), position = pos) +   
  geom_sankeynode(position = pos) + 
  facet_wrap(~pl, nrow=2)
```

**Comment by pepijn-devries** [comment]
This is a trade-off of how the plotting routine for the edges is implemented. I'm not sure if this can be fixed. If I do succeed, I will report back here...

**Comment by BAndrews22** [comment]
That's great. Thanks a mil!

**Comment by iouraich** [comment]
So does that mean that ggsankeyfier does not work well when facetting in general?

**Comment by pepijn-devries** [comment]
@iouraich, No. The faceting itself is not the issue. It is the small size of the plotting area, that is a problem. And that obviously occurs sooner when you are facetting. So facetting is not a problem, as long as your plotting device is large enough.
